### DISCUSSION TITLE
O que vocês acham da metodologia csstyle?

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#183] [Discussão]
Bom, pessoal, não sei bem como conheci este site, mas achei essa metodologia muito boa! Vi que o projeto tem quase 1k de star no github, o que parece não ser uma ideia qualquer.

Queria pedir a opinão de vocês sobre essa metodologia, pois eu gostei bastante e tenho vontade de usar. Pretendo também levantar alguns pontos relavantes para discutirmos a fim de propôr novas ideias para o conhecimento de todos.

[--csstyle](http://csstyle.io/)

Links interessantes sobre:
[Link 1](http://www.richardbagshaw.co.uk/csstyle/)
[Link 2](http://schier.co/blog/2014/10/20/semantic-css-with-csstyle.html)

Obrigado!

**Comment by felipefialho** [comment]
Não conhecia, vou dar uma estudada e falo minha opinião depois. :bear:

**Comment by kandebonfim** [comment]
Achei maneirinho. Mas não ficou muito claro quais os problemas seriam resolvidos...

**Comment by felipefialho** [comment]
@kandebonfim Fiquei com essa impressão também, mas vou dar uma estudada e tentar abstrair algum conceito :)

**Comment by diegoemerique** [comment]
Parece que é para ter menas poluição de class name no html, e principalmente para te forçar a não sair do padrão, já que não tem outra forma de fazer a não ser usando os trecos dele.

**Comment by wmartins** [comment]
Achei a ideia bem bacana, visto que usar esse tipo de coisa faz com que o código fique bastante consistente e padronizado (principalmente construindo o teu código usando os mixins providos). Os pontos que eu acho negativo é que tu tá "preso" ao SCSS (para poder usar os mixins), além de ter uma curvinha de aprendizado extra (embora pequena).
